Makefile.in

```diff
@@ -92,7 +92,7 @@ distclean:
 	do (cd $$i && $(MAKE) $(MFLAGS) $@); done
 
 systemd:
-	${INSTALL} clixon-controller.service /usr/lib/systemd/system/
+	${INSTALL} clixon-controller.service $(DESTDIR)$(libdir)/systemd/system/
```

Comment:
This makes the clixon-controller.service file be installed in /usr/local/lib/systemd/system/ instead of /usr/lib/systemd/system/ which is the proper path.

Reply:
Right, I didn't notice that because we use the default ubuntu prefix, `/usr`.  I'll come up with a better patch soon